<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTdopMain</class>
 <widget class="QMainWindow" name="QTdopMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTdopMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabConnection">
       <attribute name="title">
        <string>USB Connection</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnList">
              <property name="text">
               <string>List</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnConnect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDisconnect">
              <property name="text">
               <string>Disconnect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="conStatus">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>3</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>not connected</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tblDevices">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>5</number>
            </property>
            <property name="columnCount">
             <number>10</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabControl">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1">
          <item>
           <widget class="QFrame" name="frmStdCtrl">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Doppler Carrier Frequency</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QRadioButton" name="rbtn2MHz">
                  <property name="text">
                   <string>2 MHz</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbtn4MHz">
                  <property name="text">
                   <string>4 MHz</string>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rbtn8MHz">
                  <property name="text">
                   <string>8 MHz</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,1">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>PRF</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spbPRFkHz">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>12</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>kHz</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,2,1">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>PRF</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spbPRFHz">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,2,1">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Burst Volume</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spbBurstVolume">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Sample Volume 2</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Sample Volume 1</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Position&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Groesse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spbPosSv1">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QSpinBox" name="spbSizeSv1">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>30</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="spbPosSv2">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>50</number>
                 </property>
                 <property name="value">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QSpinBox" name="spbSizeSv2">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>30</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btnSendCtrl">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRunDevice">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>START</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabData">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="vLayData">
        <item>
         <widget class="QPushButton" name="btnData">
          <property name="text">
           <string>Read Data</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="Data1"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Data2"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Data3"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="Data4"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbCntPlot">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>50</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
